asyncbleach/_tests/example_pkg/src/example_pkg/__init__.py

```diff
@@ -1 +1,5 @@
 name = "example_pkg"
+
+
+async def f(): 
+    return await 1
```

Comment:
bleach_build_py only considers files in the _async directory, so this line will never be modified